---
layout: ~/layouts/PageLayout.astro
title: 'QIIME 2 Homepage'
---
import Text from '~/components/widgets/Text.astro';
import Hero from '~/components/widgets/Hero.astro';
import Features from '~/components/widgets/Features.astro';
import FeatureItem from '~/components/widgets/FeatureItem.astro';
import Features2 from '~/components/widgets/Features2.astro';
import Steps from '~/components/widgets/Steps.astro';
import BlogLatestPosts from '~/components/widgets/BlogLatestPosts.astro';
import FAQSection from '~/components/widgets/FAQSection.astro';
import FAQItem from '~/components/widgets/FAQItem.astro';
import Stats from '~/components/widgets/Stats.astro';
import CallToAction from '~/components/widgets/CallToAction.astro';
import { getCollection } from 'astro:content';

export const distros = (await getCollection('distros')).map(({ data }) => (data));


export {/*
An explanation of what is happening in this file and where things live.

This is an MDX file. It's markdown with a bit of JSX syntax, you may wish to
install a language pack for it for your editor. That's also why there's a weird
export statement in front of this comment.

The rest of this is Astro.

"Dynamic" data is sourced from collection loaders in:
  - src/content/config.ts
    (This is where forum posts and distros are coming from.)

URLs and Navigation should stay largely here:
  - src/navigation.ts

Config for the theme is here:
  - src/config.yaml

Colors live here:
  - src/components/CustomStyles.astro

The rest is pretty standard looking.

*/}


<Hero
  actions={[
    {
      variant: 'primary',
      text: 'Quickstart',
      href: 'https://library.qiime2.org/quickstart',
      target: '_blank',
      icon: 'tabler:arrow-forward',
    },
    { text: 'Learn more', href: '/#/features' },
  ]}
>
  <Fragment slot="title">

QIIME 2 is an AI-ready **microbiome multi-omics data science** platform

  </Fragment>
  <Fragment slot="subtitle">

[Trusted](https://scholar.google.com/scholar?hl=en&as_sdt=805&sciodt=0,3&cites=6935821321202015575,17795719731389093288,9078996940723452772,6129633569390757233,261340794755367594,16903127068530972426&scipsc=&q=&scisbd=1_),
[free](https://github.com/qiime2/qiime2/blob/dev/LICENSE),
[open source](https://github.com/qiime2),
[extensible](https://develop.qiime2.org),
and [community developed](https://library.qiime2.org)
and [supported](https://forum.qiime2.org/g/q2-mods) bioinformatics.

  </Fragment>
</Hero>


<Stats
  stats={[
    {
      title: 'Citations',
      amount: '60K+',
      href: 'https://scholar.google.com/scholar?hl=en&as_sdt=805&sciodt=0,3&cites=6935821321202015575,17795719731389093288,9078996940723452772,6129633569390757233,261340794755367594,16903127068530972426&scipsc=&q=&scisbd=1_'
    },
    {
      title: 'Forum Users',
      amount: '8K+',
      href: 'https://forum.qiime2.org'
    },
    {
      title: 'Patent Applications',
      amount: '2K+',
      href: 'https://www.lens.org/lens/search/patent/list?q=qiime%20OR%20qiime2&p=0&n=10&s=_score&d=%2B&f=false&e=false&l=en&authorField=author&dateFilterField=publishedDate&orderBy=%2B_score&presentation=false&preview=true&stemmed=true&useAuthorId=false'
    },
    {
      title: 'Clinical Trials',
      amount: '50+',
      href: 'https://clinicaltrials.gov/search?term=qiime%20OR%20qiime2'
    },
  ]}
/>


<Features
  id="/features"
  tagline="Features"
  title="Our users love these features"
  subtitle="The QIIME 2 Framework (Q2F) empowers researchers to conduct reproducible biological data science through a variety of interfaces and using diverse methods.">

  <FeatureItem title='Latest Methods' icon='tabler:microscope'>
QIIME 2 integrates dozens of plugins and hundreds of bioinformatics methods, allowing users to perform cutting-edge microbiome analysis with interactive visualizations and robust statistical tools.
  </FeatureItem>

  <FeatureItem title='Automated Provenance Recording' icon='tabler:history'>
All commands, parameters, and computational environments used during analysis are automatically recorded with decentralized data provenance.
This ensures full transparency and reproducibility.
With data provenance embedded as metadata in your data and visualizations, your data won't become disconnected from the details of how it was created.
  </FeatureItem>

  <FeatureItem title='Multiple Interfaces' icon='tabler:stack-2'>
All analytic functionality is available through multiple interfaces: Python 3 and R APIs for programmatic access; a command line interface (CLI) for efficient scripting, and a graphical interface (GUI) via Galaxy for an intuitive, visual experience.
Regardless of your computational background, QIIME 2 has an interface that will enable you to work efficiently!
  </FeatureItem>

  <FeatureItem title='Extensible' icon='tabler:hexagon-plus'>
Q2F is plugin-based, making it straight-forward to extend and customize the platform.
Developers can integrate their bioinformatics methods, leveraging an extensive library of existing functionality and leaving the interfaces and provenance recording to the framework.
  </FeatureItem>

  <FeatureItem title='Accessible' icon='tabler:book-2'>
QIIME 2 is designed to be accessible to all users, from students to seasoned researchers.
It provides simplified workflows, comprehensive tutorials, and clear documentation to streamline microbiome analysis.
Visit the [QIIME 2 Library](https://library.qiime2.org) for guides, tutorials, and reference materials.
  </FeatureItem>

  <FeatureItem title='Open Source Community' icon='tabler:users'>
QIIME 2 thrives because of its community, which drives the direction of the project.
The [QIIME 2 Forum](https://forum.qiime2.org) is the hub of the community, and serves as a free resource for discussions, troubleshooting, and collaboration.
Users are encouraged to engage and share their insights.
You can learn about our expectations of community members in the [code of conduct](https://github.com/qiime2/.github/blob/main/CODE-OF-CONDUCT.md).
  </FeatureItem>
</Features>

<Features2
    tagline="Distributions"
    title="Multi-Omics"
    subtitle="The QIIME 2 Framework supports expanding -omics data types through specialized distributions."
    items={distros}>
  <Fragment slot="bg">
    <div class="absolute inset-0 bg-blue-50 dark:bg-transparent"></div>
  </Fragment>
</Features2>


<FAQSection
    tagline="FAQs"
    title="Frequently Asked Questions"
    subtitle=""
    classes={{ container: '' }}>

  <FAQItem title='How do I pronounce QIIME 2?'>
QIIME 2 is pronounced "chime two".
It is an AI-ready microbiome multi-omics data science platform that is free, open source, extensible, and community-developed.
QIIME 2 enables researchers to conduct reproducible microbiome analysis using a variety of interfaces and tools.
  </FAQItem>

  <FAQItem title='Is QIIME 2 free to use?'>
Yes!
QIIME 2 is completely free and open source, meaning anyone can use, modify, and contribute to its development under our [BSD 3-Clause License](https://github.com/qiime2/qiime2/blob/dev/LICENSE).
That includes commercial use.
  </FAQItem>

  <FAQItem title='Is it difficult to install and use QIIME 2?'>
Not at all!

* If you have QIIME 2 results and want to interact with them without installing QIIME 2, use [QIIME 2 View](https://view.qiime2.org).
* If you want to install and use QIIME 2 on your own system, follow the [installation guide](https://library.qiime2.org/quickstart).

To learn marker gene (i.e., amplicon) analysis with QIIME 2, see the [amplicon docs](https://amplicon-docs.readthedocs.io).
To learn metagenome (i.e., shotgun) analysis with QIIME 2, see the [MOSHPIT docs](https://moshpit-docs.readthedocs.io).
To learn advanced techniques for working with the QIIME 2 Framework, see [*Using QIIME 2*](https://use.qiime2.org).
To build and distribute your own plugins, see [*Developing with QIIME 2*](https://develop.qiime2.org).
As of March 2025, we're doing an extensive re-write of our documentation using [JupyterBook](https://jupyterbook.org/en/stable/intro.html) and [Diátaxis](https://diataxis.fr/) to modernize it and to improve organization and accessibility.
  </FAQItem>

  <FAQItem title='Will my data survive the test of time?'>
Yes!
All data generated by QIIME 2 is stored in .zip files.
The file extensions `.qza` and `.qzv` stand for *QIIME Zipped Artifact* and *QIIME Zipped Visualization*.
You can open them with any unzip utility, such as `unzip`.

We call these "archives", and they contain your data in the normal formats (`.fastq.gz`, `.fasta`, `.biom`, ...).
Additionally, these archives have embedded provenance which means that even without QIIME 2 you will still have precise details on how the data was generated.
There is no need to keep a centralized database (or distributed blockchain) online for 20 years: the provenance always lives with your data inside the zip file.
You can learn about the archive format [here](https://develop.qiime2.org/en/stable/framework/explanations/archives.html).
  </FAQItem>

  <FAQItem title='What makes QIIME 2 extensible? Can I customize it to fit my work?'>
The professionally engineered architecture of QIIME 2 makes it an extremely modular and customizable tool.

The *QIIME 2 Framework (Q2F)* provides the general purpose data science functionality, like data provenance tracking, and serves as the glue between *plugins* and the many *interfaces*.
The *plugins* provide all analytic functionality, and you can have as many of these as you need.
Groups of plugins that are intended to be used together are shared as *distributions*, which makes it easy to get started right away with the most common workflows.
  </FAQItem>

  <FAQItem title='How can I get help or become involved with the QIIME 2 community?'>
The [QIIME 2 Forum](https://forum.qiime2.org) is the primary resource for support and community interaction.
There you can find announcements, job postings, and technical assistance from a vibrant and active microbiome data science community.

If you're not a software developer and want to get involved in the project, see [*How to Get Involved in the QIIME 2 Community*](https://use.qiime2.org/en/stable/how-to-guides/get-involved.html).
If you are a developer and want to get involved in the project, see [our *Contributing Guide*](https://github.com/qiime2/.github/blob/main/CONTRIBUTING.md).
  </FAQItem>

  <FAQItem title='What do you mean by "AI-Ready"?'>
We mean that you'll always know how the QIIME 2 Artifacts that you're working with were generated (because of Q2F's data provenance recording), and that the steps performed to create them made logical sense (because of Q2F's Artifact Class system).
We are now working on functionality that allows users to sign and add a license to their data, so you'll always know if or how it can be used.

Garbage In, Garbage Out.
Artifacts In, Actual Intelligence Out.
  </FAQItem>

  <FAQItem title='Where can I find more QIIME 2 plugins?'>
The [QIIME 2 Library](https://library.qiime2.org) hosts a variety of plugins and tools developed by the community, expanding QIIME 2’s capabilities beyond its base distributions.
You can also refer to [*Community Contributions*](https://forum.qiime2.org/c/community-contributions/15) on the QIIME 2 Forum.
  </FAQItem>
</FAQSection>

<Text id='/funding' title='Support from federal and foundation grants' tagline='funding'>
QIIME 2 is funded in part by NIH National Cancer Institute Informatics Technology for Cancer Research grant [1U24CA248454-01](https://reporter.nih.gov/project-details/9951750), and by grant [DAF2019-207342](https://doi.org/10.37921/862772dbrrej) from the Chan Zuckerberg Initiative (CZI) DAF,
an advised fund of Silicon Valley Community Foundation (CZI grant DOI: 10.37921/862772dbrrej; funder DOI 10.13039/100014989).

QIIME 2's documentation is (mostly) built with MyST Markdown, which is supported in part with [funding](https://sloan.org/grant-detail/6620) from the Alfred P. Sloan Foundation.

Initial support for the development of QIIME 2 was provided through a [grant](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1565100) from the National Science Foundation.

</Text>



<BlogLatestPosts id="/news" title="Announcements & News">
  <Fragment slot="bg">
    <div class="absolute inset-0 bg-blue-50 dark:bg-transparent"></div>
  </Fragment>
</BlogLatestPosts>
